formats/hocon/src/test/kotlin/kotlinx/serialization/hocon/HoconMapTest.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2017-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.serialization.hocon
+
+import com.typesafe.config.ConfigFactory
+import kotlinx.serialization.Serializable
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+private const val CONFIG_FOR_ROOT_MAP = """
```

Comment:
Please embed it into the test class to have a single classfile. Don't insist tho